src/mem/epoch/markable_atomic.rs

```diff
@@ -48,32 +53,53 @@ fn opt_owned_into_usize<T>(ptr: Option<Owned<T>>, mark: usize) -> usize {
     raw as usize | mark
 }
 
-impl<T> MarkedAtomic<T> {
-    /// Create a new, null atomic pointer.
+impl<T> MarkableAtomic<T> {
+    /// Create a new, null markable atomic pointer with a marker value of 0.
     #[cfg(feature = "nightly")]
-    pub const fn null() -> MarkedAtomic<T> {
-        MarkedAtomic {
+    pub const fn zero() -> MarkableAtomic<T> {
```

Comment:
That was would be better, but notice that for some initializers we need a `const fn`, and a function with a marker argument cannot currently be made `const` due to the `guard_mark` assertion. I don't think that feature will land anytime soon, so we need both functions for now.